src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2

```diff
@@ -313,7 +313,7 @@ module "efs_mount__{{ efs_file_system_mount.efs_name }}__{{ az }}" {
 {% endfor %}
 {% endfor %}
 
-{%- for efs_file_system_route53 in efs_file_systems %}{% if not loop.first %}, {% endif %}
+{%- for efs_file_system_route53 in efs_file_systems %}
```

Comment:
Can you explain why these changes (removing a comma) were necessary for v12? I'm not sure why they would have ever worked on v11 either 🤔

Reply:
I had included a new EFS resource to validate module but I could not create it with existing code because adding comma(,) after EFS resource block in commcarehq.tf file. I got know,  `{% if not loop.first %}, {% endif %}` is not required to create multiple EFS resources. To resolve it, I am removing `{% if not loop.first %}, {% endif %}`. 

After changes, I had verified it by adding multiple EFS file-systems.